sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -789,13 +809,17 @@ public Key restoreKey(byte[] backup) {
      * call {@link KeyClient#getKey(KeyBase baseKey)} . This will return the {@link Key keys} with key material included of the specified versions.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listKeyVersions}
      *
+     * <p>It is possible to get full keys with key material for each version from this information. Loop over all the {@link KeyBase key} by page and
```

Comment:
Description is fine, But the header should stay too.
`<p><strong>Code Samples to iterate over key versions by page</strong></p>`